experimental/adaptive-tool/README.md

```diff
@@ -1,57 +1,122 @@
 # Bot Framework Adaptive Tool
-BotFramework adaptive tool is a vscode extension to help developers to improve the fficiency of handering LG/LU files.
+Bot Framework Adaptive Tool is a Visual Studio Code extension that helps developers handle LG (.lg) and LU (.lu) files efficiently.
 
+The support includes:
+- LG/LU syntax highlighting
+- LG/LU diagnostic checks
+- LG/LU completion
+- LG template and function hover
+- LG template definition
+- LG template, function, and structure suggestions
+- Dialog (.dialog) debugging
+- LG debugging
+
+# Quick Start
+## Install from the marketplace
+1. Install the [adaptive-tool](https://marketplace.visualstudio.com/items?itemName=adaptive-tool) from the [Visual Studio Code marketplace](https://marketplace.visualstudio.com/vscode).
```

Comment:
>https://marketplace.visualstudio.com/items?itemName=adaptive-tool [](start = 31, length = 65)

When is this being published?  Link is 404 right now.

Reply:
Yes. It is just a placeholder.. Maybe release it in October? @boydc2014

Reply:
@chrimc62 last time my chat with Scott and ChrisMullins, is we will release this along with the patch release, expect in Octoboer